artman/config/converter.py

```diff
@@ -80,7 +80,7 @@ def _calculate_rel_gapic_output_dir(language, api_name, api_version):
     ouput dir becomes configurable. This logic doesn't work for non-cloud.
     """
     if language == 'java':
-        return 'java/gapic-google-cloud-%s-%s' % (api_name, api_version)
+        return 'java'
```

Comment:
Can you comment here that gapic-generator will take care of the API-specific directory name?

Reply:
@andreamlin thanks for reviewing. I actually have some more changes coming up the these two PRs so please hold off :)